data/ioc/domain.json

```diff
@@ -0,0 +1 @@
+["evil.com"]
```

Comment:
ok, so the definition is a yaml file, but now we are using a json for this part, is that confusing? or should this be:

```
- evil.com
- foo.com
- bar.com
...
```

to create a list?

Reply:
or in this case, since we are not really introducing any code or any complexities, isn't it enough to have this as a text file??? 

domain.txt with one line per domain

```
evil.com
evilfoo.com
evilbar.com
...
```

instead of building a json list, which will be long BTW... so a 5k file would be 
```
["evil.com", "evil1.com", "evil2.com", .... (5k entries)]
```

that is a single line with potentially 5k or even more domains.... depending on the amount of IOCs you are comparing against.